src/App.tsx

```diff
@@ -1,7 +1,39 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {};
+
+interface State {
+  inputKey: string
+}
+
+export class App extends React.Component<Props, State> {
+  state = {
+    inputKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKey);
+  }
+
+  handleKey = (event: KeyboardEvent) => {
+    this.setState({ inputKey: event.key });
+  };
+
+  render(): React.ReactNode {
+    const { inputKey } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          {inputKey
+            ? (`The last pressed key is [${inputKey}]`)
+            : ('Nothing was pressed yet')}
```

Comment:
parenthesis are generally redundant
```suggestion
            ? `The last pressed key is [${inputKey}]`
            : 'Nothing was pressed yet'}
```